internal/services/fingerprint/consumer.go

```diff
@@ -137,7 +209,7 @@ func (c *Consumer) Handle(ctx context.Context, event *Event) error {
 		return err
 	}
 
-	c.logger.Info().Msgf("Issued VIN credential for vehicle %d using device %s.", vn.TokenID, addr)
+	c.logger.Info().Str("device-addr", event.Subject).Msg("issued vin credential")
```

Comment:
The reason, by the way, that I'm using logger fewer fields is that Elastic starts dropping messages because the field count in the day index gets too high. I think it's possible to get around this with configuration but I am way too lazy.